javascript-practice/src/js/utils/dom.js

```diff
@@ -29,4 +29,13 @@ function createNewElement({
   return element;
 }
 
-export { createNewElement }
+/**
+ * Gets an HTML element by its ID
+ * @param {string} id - The ID of the HTML element
+ * @returns {Element} The HTML element with the specified ID
+ */
+function getEl(id) {
```

Comment:
Don't use the acronym

Should be getElementById, because you have more same function such as getElementByClass